sphinxcontrib/confluencebuilder/publisher.py

```diff
@@ -283,6 +283,7 @@ def storePage(self, page_name, raw_data, parent_id=None):
                         newPage['ancestors'] = [{'id': parent_id}]
 
                     rsp = self.rest_client.post('content', newPage)
+                    uploaded_page_id = rsp['id']
```

Comment:
Had to add this in, twas a bug in the function where it wouldn't return the uploaded_id in this scenario.